File: java/src/main/java/br/com/vps/consulting/b2b/management/partner/infrastructure/api/response/PartnerCreatedResponse.java
package br.com.vps.consulting.b2b.management.partner.infrastructure.api.response;

import io.swagger.v3.oas.annotations.media.Schema;

import java.util.UUID;

public record PartnerCreatedResponse(
        @Schema(description = "ID único do parceiro criado")
        UUID id
) {}

File: kotlin/src/main/kotlin/br/com/vps/consulting/b2b/management/partner/infrastructure/api/response/PartnerCreatedResponse.kt
package br.com.vps.consulting.b2b.management.partner.infrastructure.api.response

import io.swagger.v3.oas.annotations.media.Schema
import java.util.UUID

data class PartnerCreatedResponse(
    @field:Schema(description = "ID único do parceiro criado")
    val id: UUID,
)
